docs/cli.md

```diff
@@ -42,7 +42,9 @@ $ streamlit run --help
 .. tip::
   If you want to permanently set certain config options, just add them to
   `$CWD/.streamlit/config.toml` or to a global
-  `~/.streamlit/config.toml`. More info below.
+  `~/.streamlit/config.toml`. (in Windows this would be
+  `%userprofile%/.streamlit/config.toml`
```

Comment:
Looks like it's missing closing parentheses. Also the period should move to the end.

Reply:
thanks